### PR TITLE
RavenDB-13386 Server Certificate Replacement error

### PR DESCRIPTION
**Pull request by iftahbe**

### CODE REVIEW

File: src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs [hunk 1]
```diff
@@ -1036,6 +1039,116 @@ public async Task ReplaceClusterCert()
             RedirectToLeader();
         }
 
+        [RavenAction("/admin/certificates/localstate", "GET", AuthorizationStatus.Operator)]
```
Comment (ppekrol) [inline src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs]: local-state

File: src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs [hunk 2]
```diff
@@ -1036,6 +1039,116 @@ public async Task ReplaceClusterCert()
...
+                {
+                    foreach (var cert in certificateList)
+                        cert.Value?.Dispose();
+                }
+            }
+
+            return Task.CompletedTask;
+        }
+
+        [RavenAction("/admin/certificates/localstate/delete", "POST", AuthorizationStatus.ClusterAdmin)]
```
Comment (ppekrol) [inline src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs]: local-state and change the action to `DELETE` ?

File: src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs [hunk 3]
```diff
@@ -1036,6 +1039,116 @@ public async Task ReplaceClusterCert()
...
+                {
+                    ServerStore.Cluster.DeleteLocalState(ctx, localStateKeys);
+                    tx.Commit();
+                }
+            }
+
+            return NoContent();
+        }
+
+        [RavenAction("/admin/certificates/localstate/apply", "POST", AuthorizationStatus.ClusterAdmin)]
```
Comment (ppekrol) [inline src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs]: local-state

File: src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs (hunk 2 above)
Comment (iftahbe) [reply src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs]: yes

**Review by ppekrol — APPROVED** [review]

**Review by ayende — APPROVED** [review]

**Pull request merged**